src/tools/loadbal.py

```diff
@@ -1,4 +1,5 @@
-#  Copyright (C) 2017,2018(1H)
+#  Copyright (C) 2012,2013,2017,2018,2020(1H)
+#      Jozef Stefan Institute 
```

Comment:
I think you need a new line as `Jozef Stefan Institute` was involved in  pre 2020.